docs/book/src/topics/secondary-load-balancer.md

```diff
@@ -0,0 +1,31 @@
+# Enabling a Secondary Control Plane Load Balancer
+
+## Overview
+
+It is possible to use a second control plane load balancer within a CAPA cluster.
+This secondary control plane load balancer is primarily meant to be used for internal cluster traffic.
+
+A secondary control plane load balancer is _not_ created by default.
+The secondary control plane load balancer _must_ be a [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html).
+The secondary control plane load balancer must also be provided a name.
+
+The secondary load balancer will use the same security group information as the primary control plane load balancer.
+
+## Creating a secondary load balancer
+
+To make CAPA create a secondary load balancer, add the `secondaryControlPlaneLoadBalancer` stanza to your `AWSCluster`.
+
+```yaml
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta2
+kind: AWSCluster
+metadata:
+  name: test-aws-cluster
+spec:
+  region: us-east-2
+  sshKeyName: nrb-default
+  secondaryControlPlaneLoadBalancer:
+    name: internal-apiserver
+    loadBalancerType: nlb
```

Comment:
Can we make this the default for the secondary?

Reply:
Yeah, that makes sense. Would that go in the webhook? Because right now the defaults are set via kubebuilder annotations on the types. Not sure it's worth making a new type for the secondary LB's spec.

Reply:
Yes defaulting webhook